requirements.txt

```diff
@@ -1,2 +1,2 @@
-falcon==1.4.1
-pytelegraf==0.3.1
\ No newline at end of file
+falcon==2.0.0
```

Comment:
Why do you get rid of 1.4.1 support?
Was it tested with falcon==2.0.0?
Please, if it works with both versions allow for `>=1.4.1;<3`